widget/charts/src/components/BarChart/BarChart.tsx

```diff
@@ -60,6 +60,7 @@ export const BarChart = (props: BarChartPropTypes) => {
     buckets,
     margin = DEFAULT_MARGIN,
     getLabel,
+    isDarkTheme = false,
```

Comment:
Can't we handle the dark theme inside the chart itself?

Reply:
I test this but I didn't find any solution to handle it.